etc/alignak.backend-import.cfg

```diff
@@ -0,0 +1,210 @@
+# Alignak configuration file for importing data in the Alignak backend
```

Comment:
In the default installed configuration, it semt a good idea to provide this file as a sample of what is to be configured when using the Alignak backend...